Assets/VRTK/Source/SDK/Base/SDK_DescriptionAttribute.cs

```diff
@@ -138,10 +141,17 @@ public SDK_DescriptionAttribute(Type typeToCopyExistingDescriptionFrom, int inde
 
         public static SDK_DescriptionAttribute[] GetDescriptions(Type type)
         {
+#if UNITY_WSA && !UNITY_EDITOR
+            return type.GetTypeInfo().GetCustomAttributes(typeof(SDK_DescriptionAttribute), false)
```

Comment:
I'd prefer any `#if-#else-#endif` to only surround the one line that changed to prevent duplicated code getting out of sync between the two branches.

Here only the first line seems to be different.

Reply:
It is only the first line, however it's part of an entire Linq query, which could be different in the future.  So better to encapsulate the whole linq query

Reply:
if I can get the extension idea to work, it should be a lot cleaner.  But in my experience, UWP changes a lot of the rules sometimes and may need a different path.  Hopefully not in this case.